plugins/module_utils/mh/exceptions.py

```diff
@@ -6,17 +6,13 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
+from ansible.module_utils.common.text.converters import to_native
 
-class ModuleHelperException(Exception):
-    @staticmethod
-    def _get_remove(key, kwargs):
-        if key in kwargs:
-            result = kwargs[key]
-            del kwargs[key]
-            return result
-        return None
 
-    def __init__(self, *args, **kwargs):
-        self.msg = self._get_remove('msg', kwargs) or "Module failed with exception: {0}".format(self)
-        self.update_output = self._get_remove('update_output', kwargs) or {}
+class ModuleHelperException(Exception):
+    def __init__(self, msg, update_output=None, *args, **kwargs):
+        self.msg = to_native(msg)
```

Comment:
The default fallback for `msg` is no longer here. Shouldn't this be
```suggestion
        self.msg = to_native(msg or "Module failed with exception: {0}".format(self))
```
?